docs/source/conf.py

```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.13'
+version = '0.14'
```

Comment:
We should be able to get this into 0.13 I believe. Were these changes originally based on 0.14? Might want to back port your changes to 0.13 if that’s the case.

Reply:
Yes, just done that